internal/checks/state_test.go

```diff
@@ -21,7 +22,7 @@ func TestChecksHappyPath(t *testing.T) {
 	loader, close := configload.NewLoaderForTests(t)
 	defer close()
 	inst := initwd.NewModuleInstaller(loader.ModulesDir(), loader, nil)
-	_, instDiags := inst.InstallModules(context.Background(), fixtureDir, "tests", true, false, initwd.ModuleInstallHooksImpl{})
+	_, instDiags := inst.InstallModules(context.Background(), fixtureDir, "tests", true, false, initwd.ModuleInstallHooksImpl{}, configs.StaticModuleCall{})
```

Comment:
Yeah, a test function would make sense for this common case.

I've tried to keep the code as 1-1 as possible to make the review easier for now and am going to avoid re-formatting where possible on this large of a PR.